src/uint/boxed/cmp.rs

```diff
@@ -10,6 +10,52 @@ use subtle::{
     Choice, ConditionallySelectable, ConstantTimeEq, ConstantTimeGreater, ConstantTimeLess,
 };
 
+impl BoxedUint {
+    /// Return `b` if `c` is truthy, otherwise return `a`.
+    #[inline]
+    pub(crate) fn select(a: &Self, b: &Self, c: ConstChoice) -> Self {
+        debug_assert_eq!(a.limbs.len(), b.limbs.len());
+        let mut limbs = vec![Limb::ZERO; a.limbs.len()];
+
+        let mut i = 0;
+        while i < limbs.len() {
+            limbs[i] = Limb::select(a.limbs[i], b.limbs[i], c);
+            i += 1;
+        }
+
+        Self {
+            limbs: limbs.into(),
+        }
+    }
+
+    /// Returns the truthy value if `self >= rhs` and the falsy value otherwise.
+    #[inline]
+    pub(crate) fn gt(lhs: &Self, rhs: &Self) -> ConstChoice {
+        let (_res, borrow) = rhs.sbb(lhs, Limb::ZERO);
+        ConstChoice::from_word_mask(borrow.0)
+    }
```

Comment:
This is already impl'd as [`BoxedUint::ct_gt`](https://docs.rs/crypto-bigint/0.6.0-pre.2/crypto_bigint/struct.BoxedUint.html#method.ct_gt). Really there's not a lot of point in supporting `ConstChoice` with `BoxedUint`, and the least-common-denominator trait API should use `Choice` instead (which provides an optimization barrier in non-`const fn` contexts, which is the only place `BoxedUint` can be used)